src/library/librarywatcher.h

```diff
@@ -77,6 +78,18 @@ class LibraryWatcher : public QObject {
   void SetRescanPaused(bool pause);
 
  private:
+  class WatchedDir : public Directory {
+   public:
+    WatchedDir() { active_ = true; }
+
+    void operator=(const Directory& other) {
```

Comment:
I think this is better with an explicit constructor instead.

Reply:
Fixed.